src/static/css/octicons.css

```diff
@@ -0,0 +1,221 @@
+@font-face {
+  font-family: 'octicons';
+  src: url('../fonts/octicons.eot?#iefix') format('embedded-opentype'),
+       url('../fonts/octicons.woff') format('woff'),
+       url('../fonts/octicons.ttf') format('truetype'),
+       url('../fonts/octicons.svg#octicons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+/*
+
+.octicon is optimized for 16px.
+.mega-octicon is optimized for 32px but can be used larger.
+
+*/
+.octicon, .mega-octicon {
+  font: normal normal normal 16px/1 octicons;
+  display: inline-block;
+  text-decoration: none;
+  text-rendering: auto;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+.mega-octicon { font-size: 32px; }
+
+.octicon-alert:before { content: '\f02d'} /*  */
```

Comment:
Given that we only use a single one of these icons, do we really need this entire file? Could we merge the necessary parts into infusion-docs.css?